index.html

```diff
@@ -3,8 +3,7 @@
 <!--[if gt IE 8]><!--><html lang="en"><!--<![endif]-->
 <head>
   <meta http-equiv="content-type" content="text/html; charset=UTF-8" />
```

Comment:
```suggestion
  <meta http-equiv="content-type" content="text/html; charset=UTF-8" />
  <meta name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover">
```
Think it should have this meta tag to ensure it scales properly on mobile devices.

Reply:
thanks dude, have updated